
--| Copyright (c) 1993-2006 University of Southern California and contributors.
--| All rights reserved.
--| Your use of this work is governed under the terms of the GNU General
--| Public License version 2.

-- This is a test control file

test_name 	 safe-call-from-rescue
test_description A class calls a routine which recurses a few times and then raises a developer exception.  The exception is rescued and the rescue clause calls another routine which calls the recursive routine and retries the exception.  System execution should terminate normally, but ends with the second developer exception.

copy_sub 	 Ace $TEST Ace
copy_raw 	 test.e  $CLUSTER test.e
compile_melted
compile_result 	 ok

execute_work	 NONE exec_output1
execute_result	 ok
compare		 exec_output1 output

test_end
